vector/src/main/java/im/vector/riotredesign/features/notifications/NotificationUtils.kt

```diff
@@ -376,7 +376,7 @@ object NotificationUtils {
         val accentColor = ContextCompat.getColor(context, R.color.notification_accent_color)
         // Build the pending intent for when the notification is clicked
         val openRoomIntent = buildOpenRoomIntent(context, roomInfo.roomId)
-        val smallIcon = if (roomInfo.shouldBing) R.drawable.icon_notif_important else R.drawable.logo_transparent
+        val smallIcon = R.drawable.ic_status_bar
```

Comment:
no more bing icon then?

Reply:
nop :/, we could use a kind of level maybe?